FirebaseAuth/Sources/Auth/FIRAuth.m

```diff
@@ -482,7 +489,14 @@ - (nullable instancetype)initWithAPIKey:(NSString *)APIKey appName:(NSString *)a
     }
     UIApplication *application = [applicationClass sharedApplication];
 
-    [GULAppDelegateSwizzler proxyOriginalDelegateIncludingAPNSMethods];
+    // Use Multicast App Delegate when available but fallback to App Delegate Swizzler for backward compatibility.
+    id<GULMulticastAppDelegateProtocol> multicastDelegate =
```

Comment:
Why is this in iOS specific code?

Reply:
My guess is it's due to phone auth not being available on other platforms, but that was out of ease-of-implementation not any product decisions.